### DISCUSSION TITLE
commandlog table dropped and not re created

### DISCUSSION THREAD

**Discussion by pezzar** [dataplat/dbatools#7768] [General]
Im running the install-maintenance solution cmd and using replace existing, the commandlog table is getting dropped. 
Apart from the fact it potentially loses history data, it prevents installed jobs from running.

What is the correct sequence here please to 

A\ avoid losing history and the table
B\ ensure all components (tables, SPs, Jobs, etc) are re created

Current modus operandi doesnt seem too user friendly

**Comment by niphlod** [comment]
ahem ..... from https://docs.dbatools.io/Install-DbaMaintenanceSolution

ReplaceExisting : If this switch is enabled, objects already present in the target database will be dropped and recreated.


if you want to update the solution, use https://docs.dbatools.io/Update-DbaMaintenanceSolution ^_^
